src/esp/assets/ResourceManager.h

```diff
@@ -1110,6 +1077,18 @@ class ResourceManager {
   gfx::ShaderManager shaderManager_;
 
   // ======== Physical parameter data ========
+  /**
+   * @brief Plugin Manager used to instantiate importers which in turn are used
+   * to load asset data
+   */
+  Corrade::PluginManager::Manager<Importer> importerManager_;
+
+  /**
+   * @brief Importer used to synthesize Magnum Primitives.  This object allows
+   * for similar usage to File-based importers, but requires no file to be
+   * available/read.
+   */
+  Corrade::Containers::Pointer<Importer> primImporter;
```

Comment:
```suggestion
  Corrade::Containers::Pointer<Importer> primImporter_;
```

Underscore here also? IIRC all privates + protected vars have them.